net/nft-qos/files/nft-qos.config

```diff
@@ -73,6 +76,15 @@ config default default
 #	option ipaddr 'ABCD:FFED::1/64'
 #	option rate '1024'
 #
+# For Mac address based traffic control Samples :
+#
+# config tcrate
+#  option drunit 'kbytes'
+#  option urunit 'kbytes'
+#  option hostname 'tvbox'
+#  option macaddr '00:00:00:00:00:00'
+#  option drate '300'
+#  option urate '30'
```

Comment:
Please fix spaces and use tabs

Reply:
Hi Florian, Thanks for the comment. Please see fixed version in new commit. Thanks.